index.html

```diff
@@ -547,21 +547,22 @@
 Note that the edge cannot be brighter than the standard Schlick term, but this is generally true in real metals. We consider this a benefit of this parametrization, as it makes it impossible to produce physically implausible metals with excessively bright edges.
 
-Metal params                        | Label      | Type     | Range        | Default             | Description
-------------------------------------|------------|----------|:------------:|:-------------------:|----------------------------------------------
-**`base_weight`**                   | Weight     | `float`  | $ [0, 1]   $ | $ 1               $ | Scalar multiplier to **`base_color`**
-**`base_color`**                    | Color      | `color3` | $ [0, 1]^3 $ | $ (0.8, 0.8, 0.8) $ | Color of Fresnel reflection albedo at normal incidence, $\mathbf{F}_0$
-**`specular_weight`**               | Weight     | `float`  | $ [0, 1]   $ | $ 1               $ | Overall multiplier of the metallic Fresnel
-**`specular_color`**                | Color      | `color3` | $ [0, 1]^3 $ | $ (1, 1, 1)       $ | Tint color of metallic Fresnel reflection albedo at near-grazing incidence (i.e. around silhouettes)
-**`specular_roughness`**            | Roughness  | `float`  | $ [0, 1]   $ | $ 0.3             $ | Roughness of NDF of conductor BRDF $f_\mathrm{conductor}$
-**`specular_roughness_anisotropy`** | Anisotropy | `float`  | $ [0, 1]   $ | $ 0               $ | Anisotropy of NDF of conductor BRDF $f_\mathrm{conductor}$
+Metal params                        | Label      | Type     | Range           | Norm          | Default             | Description
+------------------------------------|------------|----------|:---------------:|:-------------:|:-------------------:|----------------------------------------------
+**`base_weight`**                   | Weight     | `float`  | $ [0, 1]      $ |               | $ 1               $ | Scalar multiplier to **`base_color`**
+**`base_color`**                    | Color      | `color3` | $ [0, 1]^3    $ |               | $ (0.8, 0.8, 0.8) $ | Color of Fresnel reflection albedo at normal incidence, $\mathbf{F}_0$
+**`specular_weight`**               | Weight     | `float`  | $ [0, \infty) $ |  $ [0, 1] $   | $ 1               $ | Overall multiplier of the metal Fresnel
+**`specular_color`**                | Color      | `color3` | $ [0, 1]^3    $ |               | $ (1, 1, 1)       $ | Tint color of metallic Fresnel reflection albedo at near-grazing incidence (i.e. around silhouettes)
+**`specular_roughness`**            | Roughness  | `float`  | $ [0, 1]      $ |               | $ 0.3             $ | Roughness of NDF of BRDF $f_\mathrm{conductor}$
+**`specular_roughness_anisotropy`** | Anisotropy | `float`  | $ [0, 1]      $ |               | $ 0               $ | Anisotropy of NDF of BRDF $f_\mathrm{conductor}$
```

Comment:
If the BRDF adjective is removed here but not in the other places, it makes the wording a bit inconsistent. Both are fine, I just have a slight preference for the verbose version.
<img width="890" alt="image" src="https://github.com/user-attachments/assets/debc008a-a882-41cb-a674-6cb945266ad8">
<img width="886" alt="image" src="https://github.com/user-attachments/assets/44e27408-ea81-4a86-92b1-b1a7e0ec5f93">